packages/editable-html/src/plugins/table/index.jsx

```diff
@@ -85,7 +85,7 @@ TableCell.propTypes = {
   ]).isRequired
 };
 
-export default opts => {
+export default (opts, imagePlugin, mathPlugin) => {
```

Comment:
The only thing the `TablePlugin` needs is the plugins array, it doesn't need the `imagePlugin` and `mathPlugin` explicitly. To that end: 
```suggestion
export default (opts, toolbarPlugins /* :  {toolbar: {}}[] */) => {
```